python/conversion/format.py

```diff
@@ -1,7 +1,9 @@
 from pathlib import Path
 
-metavision_found = True
+metavision_found_2_0 = True
```

Comment:
we should rename `metavision_found_2_0` to something more expressive. My proposal: `dat_conversion_possible`